tests/CMakeLists.txt

```diff
@@ -12,6 +12,11 @@ else()
     message("ENABLE_FULL_LIBCXX_TESTS not set - building some libcxx tests")
 endif()
 
+if(WIN32_SIMULATION)
+  message("WIN32_SIMULATION set - skipping the following group of tests:\n"
+          "abortStatus, ecall, mbed, thread, thread_local, thread_local_no_tdata")
```

Comment:
No need to specify the list of tests being skipped. How about? "WIN32_SIMULATION set and failing tests being skipped"

Reply:
Sounds good.